articles/node-vs-python/index.md

```diff
@@ -0,0 +1,179 @@
+
+### Node.js vs Python: The Differences
+- [Performance and Architecture](#performance-and-architecture)
+- [Syntax and Ease of Use](#syntax-and-ease-of-use)
+- [Type of Projects](#type-of-projects)
+- [Frameworks and Expandability](#frameworks-and-expandability)
+- [Package Management](#package-management)
+
+#### Performance and Architecture
+Python and Node.js are interpreted languages. Therefore, they are comparatively slower than compiled languages like C++.
+
+Python is a single threaded and single flowing language. Though it is optimized by frameworks such as Django, it is slow. Python runs well on servers, but is quite unsuitable for mobile. The limit in processing power is a bottleneck to Python's performance.
+
+Node.js has the upper hand here. Node.js is heavily optimized by Google's V8 JavaScript Engine, making it performant. Node.js follows an event loop based architecture, which makes it non-blocking. Asynchronous requests can be handled. This means that the runtime can execute multiple requests without getting blocked.
+
+#### Syntax and Ease of Use
+Python has extremely readable syntax. This makes it the preferred programming language for beginners to work on. It omits the use of curly braces and uses indentation to define scope.
+
+Node.js follows the JavaScript syntax, which is harder to understand than Python. But for programmers that are familiar with C/C++, Node.js syntax is similar and poses no challenge.
+
+#### Type of Projects
+Both Python and Node.js are used for web development and full-stack projects.
+
+Python is heavily used as a part of Data Science and Machine Learning projects. It has libraries like numpy, pandas, scikit, opencv and tensorflow that make it perfect for:
+
+- Neural Networks
+- Face Recognition
+- Data Processing
+
+Python is also preferred for small, lightweight websites. Flask, the micro-framework and Jinja, the templating engine is used by web developers.
+
+Due to the event loop architecture of Node.js, it is used for websites that involve a lot of concurrent processes. It is also suitable for Internet of Things solutions and projects.
+
+#### Frameworks and Expandability
+Python uses an interpreter called CPython that improves the performance of Python. There are many frameworks and libraries for full stack development.
+- [Django](https://www.djangoproject.com/) - A python web framework based on the Model-View-Controller Architecture
+- [Flask](https://flask.palletsprojects.com/en/1.1.x/) - A python micro-framework for web development
+- [Tensorflow](https://www.tensorflow.org/) - A python based Machine Learning platform
```

Comment:
There are more interpreters than just CPython. Also, I would expound on the pros/cons of each of the frameworks. Also, you should think about adding a blurb on what a micro-framework is vs a framework.